lib/lang/latin/latin-language-dataset.js

```diff
@@ -368,8 +384,29 @@ export default class LatinLanguageDataset extends LanguageDataset {
     return this
   }
 
+  getVerbsIrregularLemma (verbForm) {
+    let res = this.pos.get(Constants.POFS_VERB).types.get(Form).items.map(item => { return { val: item.value, lemma: item.features[Feature.types.hdwd].value, fullForm: item.features[Feature.types.fullForm].value } })
+    let res1 = {}
```

Comment:
Can we give `res` and `res1` more meaningful names that will reflect what data they contain? This way code will be easier to read. Thanks!

Reply:
Oh, yes - let's do it.